admin/services/data-access/sql.service.js

```diff
@@ -146,6 +146,9 @@ function parseResults (results) {
 /** SQL Service **/
 const sqlService = {}
 
+// Name of the admin database
+sqlService.adminDb = '[mtc_admin]'
```

Comment:
this is the schema, it lives within the database.
probably just rename the property/field to `adminSchema`

Reply:
Property now renamed to `adminSchema`